lib/json/static.nit

```diff
@@ -152,56 +146,36 @@ redef class FlatText
 	end
 end
 
-redef class Int
-	super Jsonable
-end
-
-redef class Float
-	super Jsonable
-end
-
-redef class Bool
-	super Jsonable
-end
-
 # A map that can be translated into a JSON object.
-interface JsonMapRead[K: String, V: nullable Jsonable]
+interface JsonMapRead[K: String, V: nullable Serializable]
 	super MapRead[K, V]
-	super Jsonable
+	super Serializable
 end
 
 # A JSON Object.
 class JsonObject
```

Comment:
I'm not fan of the object Class having a VT used only by one subclass.

Also, I'm really not ready to write `HashMap[String, nullable Serializable]` everywhere. In fact, writing it once in this message was enough. Never again.

Let's keep it this way and reassess in 6 months.